components/Premium/BuyPremium/BuyPremium.tsx

```diff
@@ -50,11 +82,29 @@ function BuyPremium() {
                 <Modal.Title>Confirmation</Modal.Title>
             </Modal.Header>
             <Modal.Body>
-                <p>
-                    You are purchasing {purchasePremiumDuration} {purchasePremiumDuration === 1 ? 'month' : 'months'} of premium for{' '}
-                    {numberWithThousandsSeperators(purchasePremiumDuration * PREMIUM_PRICE_MONTH)} CoflCoins.
-                </p>
+                <ul>
+                    <li>
+                        <span className={styles.label}>Type:</span>
+                        {purchasePremiumType.label}
```

Comment:
could the tier select be made into a toggle-button like this: https://material.angular.io/components/button-toggle/overview
and then show the extend time options in a dropdown.  
For starter I would suggest dropdown options of 
* 1 day
* 1 week
* 4 weeks
* 6 months